src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -159,14 +159,12 @@ public bool IsVisible
         public bool ShouldSwapAutomatically { get; }
 
         /// <inheritdoc />
-        public unsafe VideoMode VideoMode
+        public VideoMode VideoMode
         {
             get
             {
                 var monitor = Monitor;
-                return monitor != null
-                    ? monitor.VideoMode
-                    : _initialOptions.VideoMode;
+                return monitor?.VideoMode ?? _initialOptions.VideoMode;

```

Comment:
Could we get rid of the extra variable now that we no longer need it?

Reply:
What do you mean exactly?

Reply:
Currently we have `var monitor = Monitor`, which we no longer need now that we’re only accessing the monitor once

Reply:
Ah good point.